pkg/metrics/bootstrap.go

```diff
@@ -0,0 +1,62 @@
+package metrics
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/codes"
+
+	v1 "github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
+	"github.com/jaegertracing/jaeger-operator/pkg/tracing"
+
+	prometheusclient "github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/metric/prometheus"
+	"go.opentelemetry.io/otel/metric/global"
+	export "go.opentelemetry.io/otel/sdk/export/metric"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/histogram"
+	controller "go.opentelemetry.io/otel/sdk/metric/controller/basic"
+	processor "go.opentelemetry.io/otel/sdk/metric/processor/basic"
+	selector "go.opentelemetry.io/otel/sdk/metric/selector/simple"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+const meterName = "jaegertracing.io/jaeger"
+
+// Bootstrap prepares a new tracer to be used by the operator
+func Bootstrap(ctx context.Context, namespace string, client client.Client) {
```

Comment:
should this return some kind of closer interface to flush the spans on termination event?

Reply:
When is this method called? Periodically or just once?

Reply:
This is called just once.